nsqd/http.go

```diff
@@ -213,8 +213,8 @@ func (s *httpServer) doPUB(w http.ResponseWriter, req *http.Request, ps httprout
 
 	var deferred time.Duration
 	if ds, ok := reqParams["defer"]; ok {
-		di, err := strconv.ParseInt(ds[0], 10, 64)
-		if err != nil {
+		di, parseErr := strconv.ParseInt(ds[0], 10, 64)
+		if parseErr != nil {
```

Comment:
Are these being reported as errors in your linting?  Given the fact that an error here will return, I don't like having to complicate the naming of the error variable here.

i.e. my preference here, below and in `nsqd.go` are to keep these as they were.